app/org/sagebionetworks/bridge/models/schedules/ScheduleStrategy.java

```diff
@@ -15,7 +13,8 @@
 @JsonTypeInfo( use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 @JsonSubTypes({
     @Type(name="SimpleScheduleStrategy", value=SimpleScheduleStrategy.class),
-    @Type(name="ABTestScheduleStrategy", value=ABTestScheduleStrategy.class)
+    @Type(name="ABTestScheduleStrategy", value=ABTestScheduleStrategy.class),
+    @Type(name="CriteriaScheduleStrategy", value=CriteriaScheduleStrategy.class)
 })
```

Comment:
It uses the value of the "type" annotation to determine which class to use to deserialize (so this is one of those cases where the client has to provide the type property. Added tests to ScheduleStrategyTest to verify this works.